tests/conftest.py

```diff
@@ -16,6 +16,14 @@ def __init__(self):
             stderr=stderr,
         )
 
+    def get_stdout(self, strip_nulls=True):
+        self.stdout.seek(0)
+        payload = self.stdout.read()
+        # FIXME: There are a bunch of null-bytes at the beginning of STDOUT. Why?
```

Comment:
mhh, strange. Not sure why... i think the FIXME tag is enough to remind us later.